src/bundle/Resources/public/js/scripts/admin.trash.list.js

```diff
@@ -55,9 +55,17 @@
         const isEmptySelection = checkboxes.every((el) => !el.checked);
         const isMissingParent = checkboxes.some((el) => el.checked && parseInt(el.dataset.isParentInTrash, 10) === 1);
 
-        buttonRestore.disabled = isEmptySelection || isMissingParent;
-        buttonDelete.disabled = isEmptySelection;
-        buttonRestoreUnderNewParent.disabled = isEmptySelection;
+        if (buttonRestore) {
+            buttonRestore.disabled = isEmptySelection || isMissingParent;
+        }
+
+        if (buttonRestore) {
+            buttonDelete.disabled = isEmptySelection;
```

Comment:
Can't it be in one `if`?

Reply:
Actually, I wanted to ask about it. I made 3 `if`s because I didn't want to assume that all buttons are present or none. If I change it to `if (buttonRestore && buttonDelete && buttonRestoreUnderNewParent) {` then if only one is not present, the other 2 wont work properly. 

What do you thing about it?

Reply:
When I opened it for review, there was: `if (buttonRestore) {` three times. Now it's fixed and it makes sense.

Reply:
Oh, you're right, I have realised that I made slip and I removed `Ready for review` label but I was too slow. 😄